docs/docs/concepts/tool_calling.mdx

```diff
@@ -0,0 +1,133 @@
+# Tool Calling
+
+## Motivation 
+
+Many AI applications interact directly with humans (e.g., chatbots). In these cases, it is approrpiate for models to respond in natural langague. 
+
+But what about cases where we want a model to interact *directly* with another system (e.g., a databases or an API)?
+
+These systems often have a particular input schema (e.g., APIs have a required payload structure), making natural language inapporpiate for use.
+
+This motivates the concept of tool calling:
+
+* Tools (e.g., a databases or an API) can be bound to models.
+* Models can respond in the input schema required to run the tool.
+
+## Basic Usage
+
+:::info
+You will sometimes hear the term `function calling`. We use this term interchangeably with `tool calling`. 
+:::
+
+When using tools, we need to specify two things:
+
+* A model that supports tool calling.
+* Tools that the model can call.
+
+#### Model
+
+While tool calling is not universal, many model providers support it.
+
+See our [model integration page](/docs/integrations/chat/) for a list of models that support tool calling.
+
+LangChain provides a standardized interface for tool calling that is consistent across different models.
+
+`ChatModel.bind_tools()` is a method for specifying which tools are available for a model to call. 
+
+If a model has been initialized as `llm` without tools, we can bind tools to it as a list:
```

Comment:
Do we need to be more precise about chat model vs. llm terminology? We use them interchangeably in docs, but can be confusing?

We could also opt for favoring `model`?

Reply:
this is good point that i'm also wary of -- 
1/ maybe we defer to `model` by default
2/ conceptual guide on `llm` and `chat model` call out each specifically